server/command.go

```diff
@@ -104,7 +104,9 @@ func LoadCommands() {
 		if err != nil {
 			LogCritical("Error reading command file=%s err= %v", commandConfiguration, err)
 		}
-		for _, command := range commands {
```

Comment:
in a range loop, the value variable address (here `command`) is reused for memory efficiency, so the (pseudo code) `Commands[command.name] = &command` code at the end of the loop would always register the last command struct associated with all the names (in the docker ps/doker ps all case, both would display the ps all option)